specs/windows/windows_eventlog.table

```diff
@@ -0,0 +1,23 @@
+table_name("windows_eventlog")
+description("Table for querying all recorded Windows event logs.")
+schema([
+    Column("channel", TEXT, "Source or channel of the event", required=True),
+    Column("datetime", TEXT, "System time at which the event occurred"),
+    Column("task", INTEGER, "Task value associated with the event", additional=True),
+    Column("level", INTEGER, "Severity level associated with the event", additional=True),
+    Column("provider_name", TEXT, "Provider name of the event"),
+    Column("provider_guid", TEXT, "Provider guid of the event"),
+    Column("eventid", INTEGER, "Event ID of the event", additional=True),
```

Comment:
this is an index column, isn't it?

Reply:
Kinda want @theopolis to weigh in. I think anything that might be queried, should be an index column, so sqlite does the right thing. Table scans here are going to be really expensive.

Reply:
This is fine as additional.